test/testdata/container_config_seccomp.json

```diff
@@ -65,7 +65,7 @@
 				"user": "system_u",
 				"role": "system_r",
 				"type": "svirt_lxc_net_t",
-				"level": "s0:c4-c5"
+				"level": "s0:c4,c5"
```

Comment:
These are actually the same, but this is fine.  SELinux supports ranges c4-c5 as well as comma separated list c4,c5